documentation/getting-started-intellij-track/testing-scala-in-intellij-with-scalatest.markdown

```diff
@@ -0,0 +1,67 @@
+# Testing Scala in IntelliJ with scalatest
+
+There are multiple libraries and testing methodologies for Scala,
+but in this tutorial, we'll demonstrate one popular option from the scalatest framework
+called [FunSuite](http://www.scalatest.org/getting_started_with_fun_suite).
+We assume you know [how to build a project in IntelliJ](https://linktotutorial).
+
+## Setup
+1. Create an sbt project in IntelliJ.
+* Add the scalatest dependency to your build.sbt file:
+```
+libraryDependencies += "org.scalatest" %% "scalatest" % "3.0.1" % "test"
+```
+    * this will cause sbt to pull down the scalatest library
```

Comment:
indent is not aligned, also in some flavors of markdown it needs an extra linebreak to end code mode